src/target/riscv/riscv.h

```diff
@@ -414,4 +414,7 @@ semihosting_result_t riscv_semihosting(struct target *target, int *retval);
 void riscv_add_bscan_tunneled_scan(struct target *target, struct scan_field *field,
 		riscv_bscan_tunneled_scan_context_t *ctxt);
 
+int read_by_any_size(struct target *target, target_addr_t address, uint32_t size, uint8_t *buffer);
+int write_by_any_size(struct target *target, target_addr_t address, uint32_t size, uint8_t *buffer);
```

Comment:
Can you call this riscv_*_by_any_size(), to avoid stepping on functions that might be defined elsewhere in OpenOCD?

Reply:
done